src/parser.js

```diff
@@ -652,13 +649,41 @@ function mapClassPrivateVariables(classNode, meta) {
   return privateAssignments;
 }
 
+function mapClassPrivateCalls(classNode, meta) {
+  const privateCalls = classNode.body.expressions
+    .filter(classExpression => classExpression.constructor.name === 'Call')
+    .map(call => b.expressionStatement(mapExpression(call, meta)));
+
+  if (privateCalls.length === 0) {
+    return [];
+  }
+
+  if (!classNode.variable) {
+    throw new Error('Anonymous classes cannot contain private calls');
+  }
+
+  jsc(privateCalls).find(jsc.MemberExpression, {
```

Comment:
Unfortunately, no.  `decaf` is generating vanilla JS objects for these arguments, so looser checks are necessary.

![_mocha 2016-06-13 10-26-30](https://cloud.githubusercontent.com/assets/673655/16010933/5ab1c9d8-3151-11e6-9688-7ade2a438d6e.png)